platform-configs/optiplex-7010.robot

```diff
@@ -0,0 +1,217 @@
+${CPU_FREQUENCY_MEASURE}=                       ${TRUE}
+
+# Test module: dasharo-stab
+${TPM_DETECT_SUPPORT}=                          ${TRUE}
+${NVME_DETECTION_SUPPORT}=                      ${FALSE}
+${USB_TYPE-A_DEVICES_DETECTION_SUPPORT}=        ${TRUE}
+${NETWORK_INTERFACE_AFTER_SUSPEND_SUPPORT}=     ${TRUE}
+
+${APU_CONFIGURATION_MENU_SUPPORT}=              ${FALSE}
+${LAPTOP_EC_SERIAL_WORKAROUND}=                 ${FALSE}
+${RESET_TO_DEFAULTS_SUPPORT}=                   ${TRUE}
+${BOOT_BLOCKING_SUPPORT}=                       ${FALSE}
+${BASE_PORT_BOOTBLOCK_SUPPORT}=                 ${FALSE}
+${BASE_PORT_ROMSTAGE_SUPPORT}=                  ${TRUE}
+${BASE_PORT_POSTCAR_SUPPORT}=                   ${TRUE}
+${BASE_PORT_RAMSTAGE_SUPPORT}=                  ${TRUE}
+${BASE_PORT_ALLOCATOR_V4_SUPPORT}=              ${TRUE}
+${COREBOOT_FAN_CONTROL_SUPPORT}=                ${FALSE}
+${FAN_SPEED_MEASURE_SUPPORT}=                   ${FALSE}
+${OPTIONS_LIB}=                                 uefi-setup-menu
+${INTERNAL_LCD_DISPLAY_SUPPORT}=                ${FALSE}
+${SERIAL_NUMBER_VERIFICATION}=                  ${TRUE}
+${FAMILY_VERIFICATION}=                         ${TRUE}
+${EC_AND_SUPER_IO_SUPPORT}=                     ${FALSE}
+${EMMC_SUPPORT}=                                ${FALSE}
+${HEADS_PAYLOAD_SUPPORT}=                       ${FALSE}
+${MINI_PC_IE_SLOT_SUPPORT}=                     ${FALSE}
+${NETBOOT_UTILITIES_SUPPORT}=                   ${TRUE}
+${ETH_PERF_PAIR_2_G}=                           ${FALSE}
+${ETH_PERF_PAIR_10_G}=                          ${FALSE}
+${NVIDIA_GRAPHICS_CARD_SUPPORT}=                ${FALSE}
+${PETITBOOT_PAYLOAD_SUPPORT}=                   ${FALSE}
+${HIBERNATION_AND_RESUME_SUPPORT}=              ${TRUE}
+${PLATFORM_CPU_SPEED}=                          3.20
+${PLATFORM_RAM_SIZE}=                           16384
+${DOCKING_STATION_DETECT_SUPPORT}=              ${FALSE}
+${WATCHDOG_SUPPORT}=                            ${FALSE}
+${WIRELESS_CARD_SUPPORT}=                       ${FALSE}
+
+# Auto
+
+${INITIAL_CPU_FREQUENCY}=                       1600
+${CPU_MIN_FREQUENCY}=                           300
+${CPU_MAX_FREQUENCY}=                           3600
+${PLATFORM_RAM_SPEED}=                          800
+${CPU}=                                         Intel(R) Core(TM) i5-3470 CPU @ 3.20GHz
+${DMIDECODE_MANUFACTURER}=                      Dell Inc.
+${DMIDECODE_SERIAL_NUMBER}=                     123456789
+${DMIDECODE_PRODUCT_NAME}=                      OptiPlex 9010
+${DEF_THREADS_TOTAL}=                           4
+${DEF_THREADS_PER_CORE}=                        1
+${DEF_CORES_PER_SOCKET}=                        4
+${DEF_SOCKETS}=                                 1
+${DEF_ONLINE_CPU}=                              0-3
+${DEVICE_AUDIO1}=                               DA Intel PCH
+
+
+*** Keywords ***
+Power On
+    Sonoff Power Cycle On
+    Sleep    2s
+    Rte Power On
+
+Flash Firmware Optiplex
```

Comment:
Is this Keyword in anyway different for 7010 and 9010? If not maybe it would be a good idea to create a common config file for Optiplex machines? Same goes for ```Read Firmware Optiplex```

Reply:
Does it even work? The RteCtrl is alerady removed from this repo. We already use osfv_cli libraries. This should be enough as long as the machine is correctly supported in osfv_cli: https://github.com/Dasharo/open-source-firmware-validation/blob/develop/lib/flash.robot#L86